src/clients/java/pom.xml

```diff
@@ -233,23 +233,6 @@
               </arguments>
             </configuration>
           </execution>
-          <execution>
-            <!-- Zig build -->
-            <?m2e ignore?>
-            <id>zig-build</id>
-            <phase>validate</phase>
-            <goals>
-              <goal>exec</goal>
-            </goals>
-            <configuration>
-              <arguments>
-                <argument>build</argument>
-                <argument>java_client</argument>
-                <argument>-Drelease</argument>
-                <argument>-Dconfig=production</argument>
-              </arguments>
-            </configuration>
-          </execution>
```

Comment:
Wouldn't it be better to remove this entire `<plugin>` section and run the JNI tests from the CI as well?

Reply:
Ahh, that `test:jni` did look suspicious to me, but I failed to parse the whole thing as `zig build test:jni` invocation. XML is verbose!